src/main/java/pl/wit/projekt/JPGScanner.java

```diff
@@ -1,40 +1,83 @@
 package pl.wit.projekt;
 
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.nio.file.attribute.FileTime;
 
 public class JPGScanner {
+	
 	// Data utworzenia -> Lista nazw plików
-	private Map<Date, List<String>> mapMetaFilenames;
+	private Map<FileTime, ArrayList<Path>> filesMetadata;
```

Comment:
To pole nie jest nigdzie wykorzystywane oprócz metody getMetadata(), więc można je spokojnie usunąć.